recipes/pytorch-dev-deps/meta.yaml

```diff
@@ -0,0 +1,44 @@
+package:
+  name: pytorch-dev-deps
+  version: 1.0
+
+build:
+  noarch: python
+  number: 0
+  script:
+    - install -Dm664 {{ RECIPE_DIR }}/activate.sh {{ PREFIX }}/etc/conda/activate.d/activate-pytorch-dev-deps.sh  # [unix]
+
+requirements:
+  run:
+    - python >=3.6
+    - numpy !=1.20.0
+    - ninja
+    - pyyaml
+    - mkl
+    - mkl-include
+    - setuptools
+    - cmake
+    - cffi
+    - typing
+    - pytest
+    - compilers
+    - flake8
+    - psutil
+    - hypothesis
```

Comment:
Do we want to pin these to specific versions in any way?

Reply:
No, unless there is a specific need for a pin: we want to minimize the maintenance burden that pinning would introduce.